devtools/scripts/pyupgrad_run.py

```diff
@@ -0,0 +1,11 @@
+# clean up jupyter notebooks in main directory
+if __name__ == "__main__":
+    import subprocess
+    from pathlib import Path
+
+    files = Path("./../../weldx/").rglob("*.py")
+    for p in files:
+        subprocess.run(
+            ["pyupgrade", str(p.resolve()), "--py38-plus", "--keep-runtime-typing"],
+            check=True,
```

Comment:
```suggestion
        subprocess.run(  # skipcq: BAN-B607
            ["pyupgrade", str(p.resolve()), "--py38-plus", "--keep-runtime-typing"],
            check=True,
```

> I guess we can ignore the deepsource warning on this one, the script obviously needs pyupgrade to run 😉

That should do the trick. Maybe it needs to be in another line and I am almost certain that black won't be happy and we need a reformat 😉